pysparkling/sql/session.py

```diff
@@ -1,21 +1,20 @@
 from threading import RLock
 
-import pysparkling
-from pysparkling import RDD
-from pysparkling.context import Context
-from pysparkling.sql.conf import RuntimeConfig
-from pysparkling.sql.dataframe import DataFrame
-from pysparkling.sql.internals import DataFrameInternal
-from pysparkling.sql.readwriter import DataFrameReader
-from pysparkling.sql.schema_utils import infer_schema_from_list
-from pysparkling.sql.types import (
+from .. import Context, RDD
```

Comment:
In case circular imports are still blocking, this line might be an issue. Use `from ..context import Context` and `from ..rdd import RDD` instead.